tests/test_util.py

```diff
@@ -7,8 +7,11 @@
 
 class TestUtil(unittest.TestCase):
 
-    def test_abc(self):
-        assert dlint.util.ABC
+    def test_lendswith(self):
+        assert not dlint.util.lendswith([1, 2], [1, 2, 3])
+        assert not dlint.util.lendswith([1, 2, 3], [1, 2])
+
+        assert dlint.util.lendswith([1, 2, 3], [2, 3])
```

Comment:
I added this test because even if code isn't used in repo, it should be tested :)